docs/blog/Webots-2021-b-release.md

```diff
@@ -0,0 +1,156 @@
+Other than reducing the Webots package size to speed-up installation on servers and virtual containers, some additional Supervisor API functions to better manage the reset of the simulation has been included.
+In particular, it is now possible to save the state of a given node and all its descendant nodes and restore it later during the simulation run or export a node definition as string to duplicate the node a later point.
+Then, it is now also possible to artificially and immediately set the position of active and passive joints without having to activate the motor.
+
+An example showing the integration of OpenAI Gym with Webots is now available in our samples library.
+
+---
+
+## Reduce Package Size
+
+For this release we worked hard at finding solutions to reduce the size of the Webots installation packages.
+To this end, we removed textures, meshes and sounds from the package.
+These assets are now downloaded from the web only when needed and cached for subsequent use.
+
+%figure "New URL Format "
+![New URL Format](images/new_url_system.jpg)
+%end
+
+Additionally, we removed the offline documentation and other resources achieving the following results:
+
+
+| OS | R2021a Package Size | R2021b Package Size |
+| :---: | :---: | :---: |
+| Windows | 1.65 GB | 448 MB |
+| Linux | 1.45 GB | 263 MB |
+| macOS | 1.42 GB | 226 MB |
+
+---
+
+## New Web Rendering Engine
+
+Since the introduction of the web interface for streaming simulations and playing recorded animations, we struggled finding optimal parameters for shadows and illuminations that will reproduce exactly the same rendering as in the Webots Desktop application.
+Finally, we decided to port our custom 3D renderer WREN to WebAssembly and use it for the web simulation interface in order to minimize the differences with the Desktop version and make it straightforward to run high quality simulations and animations on the web.
+
+During this work, we also took the time to refresh and improve the graphical user interface.
+
+%figure "New Web Simulation Interface"
+![Web Simulation Interface](images/web_simulation_interface.png)
+%end
+
+---
+
+## Extended Motor and Joint Functionalities
+
+The [Motor](../reference/motor.md) and [Joint](../reference/joint.md) nodes have been extended to provide more realistic simulations.
+
+We enhanced the [Motor](../reference/motor.md) node to support coupled motors specifying the linked motor in the `name` field and setting their ratio using the new `multiplier` field.
+
+Then, we added three new PROTO nodes. The [HingeJointWithBacklash](../guide/hinge-joint-with-backlash.md) and [Hinge2JointWithBacklash](../guide/hinge-2-joint-with-backlash.md) allow users to easily add a backlash effect in hinges. The [Gear](../guide/object-gear.md) model can be used to simulate a collision-based transmission.
+
+
+%figure "Coupled Motors with Gears"
+![Coupled Motors With Gears](images/coupled_motors.gif)
```

Comment:
Just to be clear, in this gif the gears are purely visual, they're not used as a transmission. This demo is about the coupled motors where only one of the two fingers of the gripper is commanded directly and the other receives it by relay.

It's clear for me, but if it's what was assumed then it might be worth rephrasing it.

Reply:
Yes, it was also clear for me.
I think that the `Gear` PROTO is also useful just for graphical effect and not only for physical interaction.

But I will specify it so that there is no misunderstanding.